staging/src/k8s.io/cloud-provider/node/taints.go

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
maybe `staging/src/k8s.io/cloud-provider/node/helpers/taints.go` is better? Would prefer to split those funcs from "real" code.

Reply:
That's fair, let me follow up on this, thank you!